### DISCUSSION TITLE
A Chinese wiki, create a different game wiki

### DISCUSSION THREAD

**Discussion by 573077632** [requarks/wiki#5105] [Showcase your Wiki]
hello everyone. Let me say hello to Nicolas giard of wikijs first! I like wikijs and wish Mr. Nicolas giard every success in his work and life!

Many Wiki pages and many small contents need to jump many times to read. I like wikijs, which basically contains a lot of content on one page

https://wiki.ulgsy.xyz/

At present, I am alone in maintaining the wiki. If you have good suggestions, you can contact me!

**Comment by Chenghao999** [comment]
i unable to open ur URL. 
could you check ur web sever 
thx!

**Reply by ccd2s** [reply]
It looks like he's no longer maintaining it because the domains are all expired. Maybe he's busy with his life.
